model/data.go

```diff
@@ -60,3 +61,19 @@ func (g *GitCommitInfo) AddCommitMessage(msg string) {
 		g.CommitSpecifiedMsgList = append(g.CommitSpecifiedMsgList, fmt.Sprintf("fixup! %s", msg))
 	}
 }
+
+// AddCommitHash insert commit hash
+func (g *GitCommitInfo) AddCommitHash() int {
```

Comment:
It seems not to be important to store commit hash in the main process.
I think that it is divided into  two functions following as : 

-  return `headMax`
-  store commit hash when called by `qs ls` only

Could you improve this function?

Reply:
addressed.

>store commit hash when called by qs ls only

AddCommitHash is called from check current commit,
so It is always called when qs is executed.

Reply:
It seems to be fine. Thank you.